Grafana doesn't restart with Prysm Client

```
Error checking server process execution privilege. error: could not get current OS user to detect process privileges
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Starting Grafana" version=8.5.6 commit=623255cf2d branch=HEAD compiled=2022-06-14T09:25:38Z
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="The state of unified alerting is still not defined. The decision will be made during as we run the database migrations"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config loaded from" file=/usr/share/grafana/conf/defaults.ini
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config loaded from" file=/etc/grafana/grafana.ini
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from command line" arg="default.paths.data=/var/lib/grafana"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from command line" arg="default.paths.logs=/var/log/grafana"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from command line" arg="default.paths.plugins=/var/lib/grafana/plugins"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from command line" arg="default.paths.provisioning=/etc/grafana/provisioning"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from command line" arg="default.log.mode=console"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from Environment variable" var="GF_PATHS_DATA=/var/lib/grafana"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from Environment variable" var="GF_PATHS_LOGS=/var/log/grafana"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from Environment variable" var="GF_PATHS_PLUGINS=/var/lib/grafana/plugins"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Config overridden from Environment variable" var="GF_PATHS_PROVISIONING=/etc/grafana/provisioning"
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Path Home" path=/usr/share/grafana
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Path Data" path=/var/lib/grafana
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Path Logs" path=/var/log/grafana
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Path Plugins" path=/var/lib/grafana/plugins
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="Path Provisioning" path=/etc/grafana/provisioning
logger=settings t=2022-07-01T09:45:08.25+0000 lvl=info msg="App mode production"
logger=sqlstore t=2022-07-01T09:45:08.25+0000 lvl=info msg="Connecting to DB" dbtype=sqlite3
logger=sqlstore t=2022-07-01T09:45:08.25+0000 lvl=warn msg="SQLite database file has broader permissions than it should" path=/var/lib/grafana/grafana.db mode=-rwx------ expected=-rw-r-----
panic: Grafana has already been migrated to Unified Alerting.
Any alert rules created while using Unified Alerting will be deleted by rolling back.

Set force_migration=true in your grafana.ini and restart Grafana to roll back and delete Unified Alerting configuration data.

goroutine 1 [running]:
github.com/grafana/grafana/pkg/services/sqlstore/migrations/ualert.AddDashAlertMigration(0xc000c26370)
	/drone/src/pkg/services/sqlstore/migrations/ualert/ualert.go:78 +0x797
github.com/grafana/grafana/pkg/services/sqlstore/migrations.(*OSSMigrations).AddMigration(0xc000c121c0, 0xc000c26370)
	/drone/src/pkg/services/sqlstore/migrations/migrations.go:61 +0x265
github.com/grafana/grafana/pkg/services/sqlstore.(*SQLStore).Migrate(0xc000564900, 0x0)
	/drone/src/pkg/services/sqlstore/sqlstore.go:135 +0x6f
github.com/grafana/grafana/pkg/services/sqlstore.ProvideService(0xc000b90000, 0x18, {0x36f2100, 0x5763858}, {0x3717c40, 0xc000c01f40})
	/drone/src/pkg/services/sqlstore/sqlstore.go:67 +0xdc
github.com/grafana/grafana/pkg/server.Initialize({{0x7ffeb5468ce7, 0x18}, {0x7ffeb5468ccb, 0x12}, {0xc00004e0d0, 0x5, 0x5}}, {{0x0, 0x0}, {0x0, ...}, ...}, ...)
	/drone/src/pkg/server/wire_gen.go:148 +0x1b6
github.com/grafana/grafana/pkg/cmd/grafana-server/commands.executeServer({0x7ffeb5468ce7, 0x18}, {0x7ffeb5468ccb, 0x12}, {0x0, 0x0}, {0x7ffeb5468d0c, 0x6}, 0x0, {{0x3673580, ...}, ...})
	/drone/src/pkg/cmd/grafana-server/commands/cli.go:170 +0x625
github.com/grafana/grafana/pkg/cmd/grafana-server/commands.RunServer({{0x3673580, 0x5}, {0x3674ed8, 0xa}, {0x3673578, 0x4}, {0x3674ec8, 0xa}})
	/drone/src/pkg/cmd/grafana-server/commands/cli.go:107 +0x785
main.main()
	/drone/src/pkg/cmd/grafana-server/main.go:16 +0xc5
```

--- a/launcher/src/backend/ethereum-services/GrafanaService.js
+++ b/launcher/src/backend/ethereum-services/GrafanaService.js
@@ -29,7 +29,7 @@ export class GrafanaService extends NodeService {
       'bash -c "touch /etc/grafana/grafana.ini && echo \\"$GRAFANA_INI\\" > /etc/grafana/grafana.ini && /run.sh"', // command
       null, // entrypoint
       {
-        GRAFANA_INI: '[auth.anonymous]\nenabled = true\norg_role = Admin\n',
+        GRAFANA_INI: '[auth.anonymous]\nenabled = true\norg_role = Admin\nforce_migration=true\n',
       }, // env
       ports, // ports
       volumes, // volumes

